src/lib/packager/clients/pkg_finish.rb

```diff
@@ -102,6 +102,19 @@ def write
 
   private
 
+    # Checks if given source repo should be disabled
+    #
+    # Currently CD / DVD repositories are disabled after installation, when
+    # enabled in control file
+    #
+    # @param source [Symbol] if :cd or :dvd then control file is checked if the
+    #                        repo should be disabled
+    # @return [Boolean] true if the repo should be disabled
+    def disable_media_repo?(source)
+      disable_repo = ProductFeatures.GetBooleanFeature("software", "disable_media_repo")
```

Comment:
Just an optimization, we do not have to evaluate the `disable_media_repo` option if the source is not CD or DVD.

```ruby
return false unless [:cd, :dvd].include?(source)
ProductFeatures.GetBooleanFeature("software", "disable_media_repo")
```

Reply:
ok